src/psyclone/tests/psyGen_test.py

```diff
@@ -2904,6 +3014,90 @@ def test_symboltable_can_be_printed():
     assert "var2" in str(sym_table)
 
 
+def test_symboltable_specify_argument_list():
+    '''Test that the specify argument list method sets the argument_list
+    with references to each Symbol and updates the Symbol attributes when
+    needed.'''
+    symTable = SymbolTable()
```

Comment:
pylint doesn't like the capitalisation of `symTable`

Reply:
s/symTable/sym_table